pyartifactory/objects.py

```diff
@@ -752,6 +754,32 @@ def delete(self, permission_name: str) -> None:
 class ArtifactoryArtifact(ArtifactoryObject):
     """Models an artifactory artifact."""
 
+    def _walk(
+        self, artifact_path: str, topdown: bool = True, onerror=None
```

Comment:
You are right, I cloned it from `os.walk()` even though at this time it isn't being used. I was hesitating whether or not to keep it, but opted to leave it in should we ever want to need it. (For example, it might be useful at some point to add more test cases, where some files could trigger errors during attempt to download them.)
Either way, I'm not hung up on it, so if you guys would prefer not to have the unused parameter there, I can certainly remove it.